fdbclient/Schemas.cpp

```diff
@@ -157,6 +157,20 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
                         "p99.9":0.0
                      }
                   },
+                  "grv_batch_latency_statistics":{
```

Comment:
The `grv_latency_statistics` section was set up to be able to report multiple priorities. The `default` subsection refers to `default` priority, so you can move all of the contents here to a `batch` subsection there.

Reply:
ok.. It would be:
```
"grv_latency_statistics":{
  "default": { ... },
  "batch": { ... }
}
```